Highcard.rb

```diff
@@ -0,0 +1,69 @@
+require
+
+# initialize empty deck
+# build a deck
+# loop though ranks
+# .each with index -> to pull the worth of the card
+# loops suits
+# push each pair into the deck
+# shuffle the deck
+
+
+ranks = [2, 3 , 4, 5 ,6 ,7 ,8 ,9 ,10 , "A" "J" , "Q" , "K"]
+
+suits = ["hearts" , "spades" , "clubs" , "diamonds"]
+
+
+$players = []
+
+
+
+def build_deck(ranks , suits)
+
+
+deck = []
+
+ranks.each_with_index do |rank, index|
+
+suits.each do |suit|
+
+card = {
+
+suit: suit,
+rank: rank,
+worth: index
+
+}
+deck << card
+
+end
+end
+return deck.shuffle
```

Comment:
:+1:  `return` is not necessary here thanks to ruby's implicit return, but feel free to keep on using it if it helps you conceptualizing the output of this method